index.js

```diff
@@ -27,12 +27,45 @@ import { Microcopy } from './pages/Microcopy';
 import { Badges } from './pages/Badges';
 import { WidgetPages } from './pages/WidgetPages';
 import { PageText } from './pages/PageText';
+import { UXPrinciples } from './pages/UXPrinciples';
 import { DeviceBreakpoints } from './pages/DeviceBreakpoints';
 import { HomePage } from './pages/HomePage';
 import { PageNotFound } from './pages/PageNotFound';
 
 const PAGES = {
-  home: HomePage,
+  home: () => (
+    <div>
```

Comment:
This should have stayed inside the HomePage.js file, rather than be pulled into the Sidebar